src/main/java/Bundle_fr.properties

```diff
@@ -0,0 +1,2021 @@
+harvestclients.newClientDialog.schedule=P�riodicit�
+harvestclients.newClientDialog.schedule.tip=Programmer le moissonnage pour qu'il s'ex�cute automatiquement de fa�on quotidienne ou hebdomadaire.
+harvestclients.newClientDialog.schedule.time.none.helptext=Ne pas sp�cifier de p�riodicit� de moissonnage de sorte que l'ex�cution se fera sur demande seulement.
+harvestclients.newClientDialog.schedule.none=Aucun
+harvestclients.newClientDialog.schedule.daily=Quotidiennement
+harvestclients.newClientDialog.schedule.weekly=Hebdomadairement
+harvestclients.newClientDialog.schedule.time=Horaire
+harvestclients.newClientDialog.schedule.day=Jour
+harvestclients.newClientDialog.schedule.time.am=a.m.
+harvestclients.newClientDialog.schedule.time.pm=p.m.
+harvestclients.newClientDialog.schedule.time.helptext=L'horaire programm� se r�f�re � votre heure locale.
+harvestclients.newClientDialog.btn.create=Cr�er un client
+harvestclients.newClientDialog.success=Le client de moissonnage �\u00A0{0}\u00A0� a bien �t� cr��.
+harvestclients.newClientDialog.step4=�tape 4 de 4 - Affichage
+harvestclients.newClientDialog.harvestingStyle=Type de d�p�t
+harvestclients.newClientDialog.harvestingStyle.tip=Type du d�p�t distant.
+harvestclients.newClientDialog.harvestingStyle.helptext=S�lectionnez le type de d�p�t qui d�crit le mieux ce serveur distant afin d'appliquer correctement les r�gles de formatage et de style aux m�tadonn�es r�colt�es lors de leur affichage dans les r�sultats de recherche. Notez qu'une s�lection incorrecte du type de d�p�t distant peut entra�ner l'affichage incomplet des entr�es dans les r�sultats de recherche causant ainsi une impossibilit� de rediriger l'utilisateur vers le d�p�t source des donn�es.
+harvestclients.viewEditDialog.title=Modifier le client de moissonnage
+harvestclients.viewEditDialog.archiveUrl=URL du d�p�t
+harvestclients.viewEditDialog.archiveUrl.tip=L'URL du d�p�t qui fournit les donn�es moisssonn�es par ce client, laquelle est utilis�e dans les r�sultats de recherche pour les liens vers les sources originales du contenu moissonn�. 
+harvestclients.viewEditDialog.archiveUrl.helptext=Modifier si cet URL est diff�rent de l'URL du serveur.
+harvestclients.viewEditDialog.archiveDescription=Description du d�p�t
+harvestclients.viewEditDialog.archiveDescription.tip=Description du d�p�t source du contenu moissonn� et affich� dans les r�sultats de recherche.
+harvestclients.viewEditDialog.archiveDescription.default.generic=Cet ensemble de de donn�es est moissonn� aupr�s de nos partenaires. En cliquant sur le lien, vous acc�dez directement au d�p�t source des donn�es.
+harvestclients.viewEditDialog.btn.save=Sauvegarder les modifications
+harvestclients.newClientDialog.title.edit=Modifier le groupe {0}
+
+#harvestset.xhtml
+harvestserver.title=Administration du serveur de moissonnage
+harvestserver.toptip=- D�finir les collections d'ensembles de donn�es locaux qui seront disponibles pour le moissonnage par les clients distants.
```

Comment:
Ok, I will add the missing space. And now that you have explained how this snippet of text displays along with an another part, I would rather use an em dash ("—") here instead of an hyphen ("-").